### DISCUSSION TITLE
빌드잇 작업후 유니티로 가져왔을때 캐릭터 계단에 끼임 현상

### DISCUSSION THREAD

**Discussion by almondsky** [naverz/zepeto-studio-kor#987] [5. Q&A]
안녕하세요, 
빌드잇에서 건물의 전체적 외형 작업후 유니티로 가져와 캐릭터 불러오기를 실행했습니다. 
캐릭터를 여기저기 이동해보며 계단같은 높이가 있는 오브젝트에서 점프 실행시 계단을 밟고 위로 올라가는 것이 아니라 계단 속에 파묻히게 되는데,,, 왜 이런 현상이 일어나는 것일까요?? 
<img width="615" alt="스크린샷 2022-07-11 오후 11 18 12" src="https://user-images.githubusercontent.com/105544091/178286140-b42e28fe-8a78-487b-80ed-866bb1e18c2f.png">

**Comment by YSHFM** [comment]
콜라이더가 제대로 붙어 있는 게 맞을까요?

**Reply by almondsky** [reply]
콜라이더 문제였습니다,! 답변 너무 감사합니다!!
